foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -1381,14 +1381,15 @@ def monitor_jobs():
                 logger.error("Caught exception in %s: ", function.__name__)
                 traceback.print_exc(chain=False)
 
-        if timezone.now() - last_janitorial_time > JANITOR_DISPATCH_TIME:
-            send_janitor_jobs()
-            cleanup_the_queue()
-            last_janitorial_time = timezone.now()
-
-        if timezone.now() - last_dbclean_time > DBCLEAN_TIME:
-            clean_database()
-            last_dbclean_time = timezone.now()
+        if settings.RUNNING_IN_CLOUD:
```

Comment:
Why do we need this here?

Reply:
This is to prevent local jobs from being cleaned up which I had to do while testing. It could be its own PR but I kept it since it was loosely related.